ui/app/models/auth-config/github.js

```diff
@@ -1,24 +1,28 @@
 import { computed } from '@ember/object';
 import DS from 'ember-data';
-
 import AuthConfig from '../auth-config';
 import fieldToAttrs from 'vault/utils/field-to-attrs';
+import { combineFieldGroups } from 'vault/utils/openapi-to-attrs';
 
 const { attr } = DS;
 
 export default AuthConfig.extend({
+  useOpenAPI: true,
   organization: attr('string'),
   baseUrl: attr('string', {
     label: 'Base URL',
   }),
 
   fieldGroups: computed(function() {
-    const groups = [
+    let groups = [
       { default: ['organization'] },
       {
         'GitHub Options': ['baseUrl'],
       },
     ];
+    if (this.newFields) {
+      groups = combineFieldGroups(groups, this.newFields, []);
```

Comment:
this combines the fields we have defined in the model with those returned from OpenAPI. new fields should not be added to the model -- ideally we should transition to _only_ using fields from OpenAPI since it's meant to be the source of truth.